CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
```

Comment:
This should offer more explanation on how they can do this. i.e. `Added support for customization of user agent information. This can be done by setting a value for `com.rakuten.tech.mobile.miniapp.HostAppUserAgentInfo` in your manifest.` Or maybe just add a link to `Configure SDK settings in AndroidManifest.xml` section?

Reply:
I think that is out of scope of the `changelog`.
`how-to` should be a part of the UserGuide instead. What do you think?

Reply:
This also got addressed in the userguide. This will get better with the linking afterwards.